packages/core/src/components/Input/Input.stories.tsx

```diff
@@ -0,0 +1,649 @@
+                  setFirstNameErrorMessage("The first name is too short");
+                }
+              }}
+            />
+          </HvGrid>
+          <HvGrid item xs={12}>
+            <HvInput
+              label="Last name"
+              description="Please enter your last name"
+              placeholder="Insert last name"
+              defaultValue="Not a name!"
+              required
+              status={lastNameValidationState}
+              aria-errormessage="lastName-error"
+              onFocus={(value) => {
+                setLastNameValidationState(value ? "standBy" : "empty");
+              }}
+              onBlur={(_e, _value, inputValidity) => {
+                setLastNameValidationState("invalid");
+
+                if (inputValidity.valueMissing) {
+                  setLastNameErrorMessage("You must provide a last name");
+                } else {
+                  setLastNameErrorMessage(
+                    "Nice try, but the last name will always be invalid. I told you!"
+                  );
+                }
+              }}
+            />
+          </HvGrid>
+        </HvGrid>
+        <HvGrid item xs={7}>
+          <div
+            style={{
+              paddingTop: "20px",
+              paddingLeft: "30px",
+              backgroundColor: theme.colors.sema9,
+              color: theme.colors.base2,
+              height: "100%",
+            }}
+          >
+            <h4>Form errors:</h4>
+            <ul>
+              {firstNameErrorMessage && (
+                <li id="firstName-error" aria-live="polite">
+                  {firstNameErrorMessage}
+                </li>
+              )}
+              {lastNameErrorMessage && (
+                <li id="lastName-error" aria-live="polite">
+                  {lastNameErrorMessage}
+                </li>
+              )}
+            </ul>
+          </div>
+        </HvGrid>
+      </HvGrid>
+    );
+  },
+};
+
+export const NumericRequired: StoryObj<HvInputProps> = {
+  parameters: {
+    docs: {
+      description: {
+        story:
+          "Required Input that only accepts numbers and limits the maximum length to 5.",
+      },
+    },
+  },
+  render: () => {
+    const validationMessages = {
+      requiredError: "The number is required",
+      maxCharError: "Number is too big",
+      typeMismatchError: "Value is not a number",
+    };
+
+    return (
+      <HvInput
+        id="numeric-required-input"
+        type="number"
+        label="Height"
+        description="Enter a numeric value"
+        placeholder="Insert a number"
+        validationMessages={validationMessages}
+        required
+        maxCharQuantity={5}
+        showValidationIcon
+      />
+    );
+  },
+};
+
+export const Email: StoryObj<HvInputProps> = {
+  parameters: {
+    docs: {
+      description: {
+        story:
+          "Required Input that only accepts numbers and limits the maximum length to 5.",
```

Comment:
Wrong description: "Required Input that only accepts valid emails.".

Reply:
Fixed!